src/evaluations/data/evaluation_configs.py

```diff
@@ -262,24 +320,21 @@ def _generate_configs_scenario_5(num_sets, small_set_size,
   Returns:
     A list of ScenarioConfigs of scenario 5 Sequentiall correlated sets.
   """
+
   name_to_choices_of_set_size_list = {
-      '1st_small_then_large': [small_set_size] + [large_set_size] * (
-          num_sets - 1),
+      **_get_default_name_to_choices_of_set_size_list(
```

Comment:
I didn't add all of the fields to default method considering this scenario is sensitive to orders and we may want to test more cases that may not be needed in other scenarios, but let me know if that's the case